contrib/testgen/gen_base58_test_vectors.py

```diff
@@ -15,10 +15,10 @@
 from binascii import b2a_hex
 
 # key types
-PUBKEY_ADDRESS = 0
-SCRIPT_ADDRESS = 5
-PUBKEY_ADDRESS_TEST = 111
-SCRIPT_ADDRESS_TEST = 196
+PUBKEY_ADDRESS = [0x1C, 0xB8]
+SCRIPT_ADDRESS = [0x1C, 0xBD]
+PUBKEY_ADDRESS_TEST = [0x1D, 0x25]
+SCRIPT_ADDRESS_TEST = [0x1C, 0xBA]
```

Comment:
I'd prefer these to be tuples, e.g. `(0x1C, 0xB8)`. (A tuple is immutable, unlike a list.)